packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1993,20 +2081,21 @@ class FollowerLayer extends ContainerLayer {
       return null; // this does not have an engine layer.
```

Comment:
while you're here, can you hyperlink the class names in the docs to `alwaysNeedsAddToScene` above? thanks

Reply:
Done.